### PR TITLE
chore: code improvements

### PR DESCRIPTION
**Pull request by geka-evk**
chore: code improvements

### CODE REVIEW

File: src/core-connector-svc/coreConnectorRoutes.ts [hunk 1]
```diff
@@ -50,16 +49,17 @@ export class CoreConnectorRoutes implements IRoutes{
...
 
-    getRoutes(): ServerRoute<ReqRefDefaults>[] {
+    getRoutes(): ServerRoute[] {
         return this.routes;
     }
 
     private async getParties(request: Request, h: ResponseToolkit){ //todo change to openapi signature
-        this.logger.info("Received GET request /parties/{IdType}/{ID}");
+        // this.logger.info("Received GET request /parties/{IdType}/{ID}");
+        // we don't need this - loggingPlugin will log such info for any incoming requests
```
Comment (geka-evk) [inline src/core-connector-svc/coreConnectorRoutes.ts]: here's an example how logs for any requests look like:
image

Comment (elijah0kello) [reply src/core-connector-svc/coreConnectorRoutes.ts]: yeah looks great actually. Thanks @geka-evk

**Comment by elijah0kello** [comment]
I am going to pull this branch and work on it to incorporate the changes and I'll push to this branch after which we can review and merge.

**Review by elijah0kello — APPROVED** [review]
I have adapted your changes and updated the code base with your recommendations. Thanks

**Pull request merged**